src/api/spec/models/workflow/step/set_flags_step_spec.rb

```diff
@@ -7,9 +7,10 @@
 
   let(:user) { create(:confirmed_user, :with_home, login: 'Iggy') }
   let(:workflow_run) do
-    create(:workflow_run, scm_vendor: scm_vendor, hook_event: hook_event, request_payload: request_payload)
+    create(:workflow_run, scm_vendor: scm_vendor, hook_event: hook_event, hook_action: hook_action, request_payload: request_payload)
```

Comment:
```suggestion
    create(:workflow_run, scm_vendor: scm_vendor, hook_event: hook_event, hook_action: nil, request_payload: request_payload)
```
you can inline it, its not used anywhere else.